cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -2137,5 +2277,27 @@ private String getFileName (String url) {
         String[] pathElements = url.split("/");
         return pathElements[pathElements.length - 1];
     } // getFileName
+    
+    /**
+     * getDescriptionsMap: return a map with basic descriptions in a instance configuration file.
+     * 
+     */
+    private Map<String,String> getDescriptionsMap () {
```

Comment:
Do you think there is a way of reading at run time these descriptions, instead of predefining them here? The problem with this approach is any other comment the user may add by hand is not reflected anymore in the configuration once the API is used.